Sources/SKCore/BuildServerBuildSystem.swift

```diff
@@ -85,8 +110,13 @@ public final class BuildServerBuildSystem {
   }
 
   private func initializeBuildServer() throws {
-    let serverPath = AbsolutePath(serverConfig.argv[0], relativeTo: projectRoot)
-    let flags = Array(serverConfig.argv[1...])
+    let interpreter =
+        lookupExecutablePath(filename: executable("python3"), searchPaths: searchPaths)
+            ?? lookupExecutablePath(filename: executable("python"), searchPaths: searchPaths)
```

Comment:
Since the Python scripts had a shebang of `#!/usr/bin/env python3` I don’t think we need to look for the `python` executable. That will just result in Python 2 vs Python 3 syntax errors down the line.

Reply:
We do need to search for `python` as Windows uses `python.exe` always.

Reply:
I see. Makes sense 👍